patches/api/0468-Reference-InventoryDragEvent-in-InventoryClickEvent.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: Axel <axel.vivenot@outlook.fr>
+Date: Sun, 7 Apr 2024 19:17:03 +0200
+Subject: [PATCH] Referenced InventoryDragEvent in documentation of
+ InventoryClickEvent
+
+
+diff --git a/src/main/java/org/bukkit/event/inventory/InventoryClickEvent.java b/src/main/java/org/bukkit/event/inventory/InventoryClickEvent.java
+index ac67a4e321f43d1ede09dafe2daa1f07de8a923f..7126f3a9f2fd9b529be8063490f2d0bf0839aac8 100644
+--- a/src/main/java/org/bukkit/event/inventory/InventoryClickEvent.java
++++ b/src/main/java/org/bukkit/event/inventory/InventoryClickEvent.java
+@@ -16,6 +16,9 @@ import org.jetbrains.annotations.Nullable;
+ /**
+  * This event is called when a player clicks in an inventory.
+  * <p>
++ * In case of a drag action within an inventory, InventoryClickEvent is only called at the beginning of the drag.
++ * {@link InventoryDragEvent} is then called at the end of the drag.
++ * <p>
```

Comment:
Half of this isn't true. If its a drag, the click event isn't fired at all and the drag is fired at the end.